src/Compilers/Core/CodeAnalysisTest/DiffUtilTests.cs

```diff
@@ -11,7 +11,15 @@ public class DiffUtilTests
         public void TestDiffTester()
         {
             // Make sure the diff tester is working!
-            Assert.Equal(DiffUtil.DiffReport(
+            Assert.Equal(@"    A
+++> 1
+    B
+    C
+--> D
+    E
+++> 2
+--> F",
+DiffUtil.DiffReport(
```

Comment:
>DiffReport [](start = 9, length = 10)

nit: probably should be indented (since it's code)

Reply:
I ended up factoring this out into a local because I couldn't find a way to make it look present in it's current form. 

---
In reply to: [327340753](https://github.com/dotnet/roslyn/pull/38789#discussion_r327340753) [](ancestors = 327340753)